source/slang/slang-check-conformance.cpp

```diff
@@ -452,67 +252,16 @@ namespace Slang
             // witness for each of the case types, and that is
             // enough to build a witness for the tagged union.
             //
-            if(outWitness)
-            {
-                TaggedUnionSubtypeWitness* taggedUnionWitness = m_astBuilder->create<TaggedUnionSubtypeWitness>();
-                taggedUnionWitness->sub = taggedUnionType;
-                taggedUnionWitness->sup = DeclRefType::create(m_astBuilder, superTypeDeclRef);
-                taggedUnionWitness->caseWitnesses.swapWith(caseWitnesses);
+            TaggedUnionSubtypeWitness* taggedUnionWitness = m_astBuilder->create<TaggedUnionSubtypeWitness>();
```

Comment:
should this be getOrCreate?

Reply:
Probably, but at this point I think we need to go ahead and remove tagged union types from the codebase. The dynamic dispatch work does everything the tagged unions did, better (at least as far as I can remember).